ledger/participant-integration-api/src/test/lib/scala/platform/store/backend/StorageBackendTestsInitializeIngestion.scala

```diff
@@ -47,6 +48,7 @@ private[backend] trait StorageBackendTestsInitializeIngestion
       dtoPackageEntry(offset(8)),
       // 9: transaction with create node
       dtoCreate(offset(9), 4L, "#9"),
+      DbDto.CreateFilter(4L, someTemplateId.toString, someParty.toString),
```

Comment:
There are many places where we ingest a create event without a corresponding filter event. The same already happens with completion events (not every `dtoCreate` is followed by a `dtoCompletion`) - the missing DTOs are not relevant for the respective test.
I'm still slightly worried about the fact that we have an increasing number of DTO sequences that cannot be produced by a real indexer.

Reply:
The purpose of these tests is not to test the real indexer, rather some targeted functionality. These are unit tests.